Machine.Specifications.ReSharper.Provider.10/RunStrategies/MspecOutOfProcessUnitTestRunStrategy.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using JetBrains.ReSharper.TaskRunnerFramework;
+using JetBrains.ReSharper.UnitTestFramework;
 using JetBrains.ReSharper.UnitTestFramework.Strategy;
 using Machine.Specifications.ReSharperRunner;
 
 namespace Machine.Specifications.ReSharperProvider.RunStrategies
 {
-    public class MspecOutOfProcessUnitTestRunStrategy : OutOfProcessUnitTestRunStrategy
+    public class MspecOutOfProcessUnitTestRunStrategy : TaskRunnerOutOfProcessUnitTestRunStrategy
     {
-        public MspecOutOfProcessUnitTestRunStrategy()
-            : base(new RemoteTaskRunnerInfo(MspecTaskRunner.RunnerId, typeof(MspecTaskRunner)))
+        public MspecOutOfProcessUnitTestRunStrategy(IUnitTestAgentManager agentManager, IUnitTestResultManager resultManager)
+            : base(agentManager, resultManager, new RemoteTaskRunnerInfo(id: MspecTaskRunner.RunnerId, runnerType: typeof(MspecTaskRunner)))
```

Comment:
There's only 2 params for `RemoteTaskRunnerInfo`, no need to specify `id:` and `runnerType:`.